rviz_common/src/rviz_common/selection/selection_manager.cpp

```diff
@@ -125,8 +125,6 @@ SelectionManager::~SelectionManager()
   }
   delete[] reinterpret_cast<uint8_t *>(depth_pixel_box_.data);
 
-  vis_manager_->getSceneManager()->destroyCamera(camera_);
```

Comment:
Why is this needed? Is there somewhere else where this is happening instead?

Reply:
At this point the scene manager is already deleted and `getSceneManager()` returns a null pointer. Also the OgreSceneManager does delete all cameras in its destructure. Unfortunately, the Ogre documentations does not say so, but a look at the [Ogre code](https://github.com/OGRECave/ogre/blob/ea41ed9ec36ca896b695b41bee3dbda8855f0767/OgreMain/src/OgreSceneManager.cpp#L194) is necessary. Therefore the explicit camera destruction is not necessary.